assets/scss/7-utilities/_spacing.scss

```diff
@@ -0,0 +1,51 @@
+// Margin helpers (has-<size>-btm-marg)
+//
+// Experimental: Bottom margin only. Opt for these instead of adding margin-bottom to a new class.
+//
+// .has-xxxl-btm-marg -  Margin bottom with $size-xxxl
+// .has-giant-btm-marg -  Margin bottom with $size-giant
+// .has-xxl-btm-marg -  Margin bottom with $size-xxl
+// .has-xl-btm-marg -  Margin bottom with $size-xl
+// .has-l-btm-marg -  Margin bottom with $size-l
+// .has-m-btm-marg -  Margin bottom with $size-m
+// .has-b-btm-marg -  Margin bottom with $size-b
+// .has-s-btm-marg -  Margin bottom with $size-s
+// .has-xs-btm-marg -  Margin bottom with $size-xs
+// .has-xxs-btm-marg -  Margin bottom with $size-xxs
+// .has-xxxs-btm-marg -  Margin bottom with $size-xxxs
+//
+// Markup: <div style="background:#ffc200;" class="{{ className }}">Example</div>
```

Comment:
I'm wondering if there's a way to demo these that might be more apparent. For example what if the markup used a list and we add a border to show the separation. So like...
this
![Screen Shot 2019-05-01 at 1 45 20 PM](https://user-images.githubusercontent.com/2974713/57037267-54da7d80-6c1c-11e9-8b44-e99db4802c31.png)

instead of this
![Screen Shot 2019-05-01 at 1 45 42 PM](https://user-images.githubusercontent.com/2974713/57037266-54da7d80-6c1c-11e9-8aea-8655e37cdad9.png)

Reply:
I did this!

Reply:
looks lovely! Thank you